src/components/Navbar/styles.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../sass/index.scss';
+
+.navbar--transparent {
+  background-color: transparent !important;
+
+  .e-logo {
+    max-width: 7rem;
+    max-height: 7rem;
+    cursor: pointer;
+  }
+
+  .e-logo-small {
```

Comment:
Since this is a variation of logo, you can keep the name as e-logo--small and nest it inside e-logo.

Reply:
Thanks for the feedback, missed it.